content/docs/user-guide/data-management/index.md

```diff
@@ -0,0 +1,105 @@
+# Data Management with DVC
+
+Storing and transferring datasets and ML models can vary depending on project
+needs, available infrastructure, etc. DVC helps you avoid logistics like object
+permissions on cloud storage, sync tools and schedules, back up snapshots, etc.
+and focus on machine learning.
+
+You work with data normally in a local <abbr>workspace</abbr>. DVC tracks,
+restores, and synchronize everything with a few, straightforward operations that
+do not change regardless of the underlying file systems, transfer protocols,
+etc.
+
+![]() _Separating data from code (codification)_
+
+<details>
+
+## Click to learn more about data _codification_
+
+To achieve this, DVC replaces large files and directories with small [metafiles]
+that describe the assets. Data files are moved to a separate <abbr>cache</abbr>
+but kept virtually (linked) in the workspace. This separates your data from code
+(including metafiles).
+
+<admon type="tip">
+
+This also allows you to [version] project files with Git, a battle-tested [SCM]
+tool.
+
+</admon>
+
+[version]: /doc/user-guide/data-management/data-versioning
+[scm]: https://www.atlassian.com/git/tutorials/source-code-management
+
+</details>
+
+Your experience can stay consistent because DVC works [indirectly], by checking
+the [metafiles] and [configuration] of your <abbr>project</abbr> to find out
+where and how to handle files. This is transparent to you as user, but it's
+important to understand the mechanics in general.
+
+[metafiles]: /doc/user-guide/project-structure
+[indirectly]: https://en.wikipedia.org/wiki/Indirection
+[configuration]: /doc/command-reference/config
+
+## Workflow and benefits
+
+**Before**: Files are scattered in the cloud; You use low-level operations
+specific to each storage (e.g. `aws s3 cp`); Ad hoc file names are used to save
```

Comment:
it's not only aws s3 cp, it can be Python ... 

I think point here is that people (have to) access it directly, they need to manage hardcoded path ...

we need to reflect on before / after more, make it a core and explain the difference in an accessible way that it's very easy to understand it